src/components/pages/ProfilePage.js

```diff
@@ -1,21 +1,131 @@
+            <li className="flex">
+              <p className="text-white font-semibold inline py-2 w-1/6">
+                First name
+              </p>
+              <input
+                className="bg-faded-dark-gray focus:outline-none focus:shadow-outline ml-20 py-2 px-4 block w-1/3 appearance-none leading-normal shadow-md caret-white text-white"
+                type="text"
+                placeholder={user?.attributes?.given_name}
+              />
+            </li>
+            <li className="flex">
+              <p className="text-white font-semibold inline py-2 w-1/6">
+                Last name
+              </p>
+              <input
+                className="bg-faded-dark-gray focus:outline-none focus:shadow-outline ml-20 py-2 px-4 block w-1/3 appearance-none leading-normal shadow-md caret-white text-white"
+                type="text"
+                placeholder={user?.attributes?.family_name}
+              />
+            </li>
+            <li className="flex">
+              <p className="text-white font-semibold inline py-2 w-1/6">
+                Email Address
+              </p>
+              <input
+                className="bg-faded-dark-gray focus:outline-none focus:shadow-outline ml-20 py-2 px-4 block w-1/3 appearance-none leading-normal shadow-md caret-white text-white"
+                type="text"
+                placeholder={user?.attributes?.email}
+              />
+            </li>
+            <li className="flex">
+              <p className="text-white font-semibold inline py-2 w-1/6">
+                Phone Number
+              </p>
+              <input
+                className="bg-faded-dark-gray focus:outline-none focus:shadow-outline ml-20 py-2 px-4 block w-1/3 appearance-none leading-normal shadow-md caret-white text-white"
+                type="text"
+                placeholder={user?.attributes?.phone_number}
+              />
+            </li>
+            <li>
+              <ul className="grid grid-cols-1 gap-6 mt-5">
+                <li>
+                  <button className="text-white font-semibold underline">
+                    Change password
+                  </button>
+                  <FaArrowRight className="inline mb-1 ml-1 text-white" />
+                </li>
+                <li>
+                  <button className="text-white font-semibold underline">
+                    Update Alpaca Key
+                  </button>
+                  <FaArrowRight className="inline mb-1 ml-1 text-white" />
+                </li>
+                <li>
+                  <button className="text-red font-semibold underline">
+                    Delete account
+                  </button>
+                  <FaArrowRight className="inline mb-1 ml-1 text-red" />
+                </li>
+              </ul>
+            </li>
+            <li>
+              <button className="text-white font-medium rounded-lg bg-green py-2 px-6">
+                Save changes
```

Comment:
The logic for this should be done by you. Tim can hook up the stuff to Amplify but you should have it setup where its as simple has plugging in the necessary function calls to update the fields. This will require some more logic first such as storing the values in each input box or having some way of referencing the input boxes when changes are saved.

Reply:
That is beyond surface-level UI though, I can add that but I didn't think that would be a part of this story. I figured I would just be helping Tim implement that.